specification/search/data-plane/Azure.Search/stable/2024-07-01/searchservice.json

```diff
@@ -8787,12 +8787,26 @@
       "allOf": [
         {
           "$ref": "#/definitions/SearchIndexerSkill"
-        },
-        {
-          "$ref": "#/definitions/WebApiParameters"
         }
       ],
       "properties": {
+        "uri": {
+          "type": "string",
```

Comment:
One thing here I noticed, it looks like this had been changed to add "format": "uri" since that was what WebApiParameters was using. Should we still add that, or that would be considered too breaking as well?

Reply:
I'd say for back-compat, it'd be best to revert every change on this just to ensure external consumers don't have inadvertent breaks.

Reply:
The breaking change board usually approves format changes as the REST API is not broken but the SDKs may be broken by this (depends on the language).

Reply:
I wasn't completely clear on my last reply. The change here reverts back to removing `format: uri` which was added new when `WebApiSkill` began extending `WebApiParameters`. So, this will retain previous behavior where `WebApiSkill` does not have `format: uri` for this property, but `WebApiParameters` should retain that as we agreed that was the design we wanted.